pyhive/hive.py

```diff
@@ -299,7 +299,7 @@ def _fetch_more(self):
         if not new_data:
             self._state = self._STATE_FINISHED
 
-    def poll(self):
+    def poll(self, getProgressUpdate=False):
```

Comment:
> What's the purpose in having this be an option instead of just always true? (or default true with option to turn it off for exceptional cases)

Mmm, i thought I should not change the default behaviour...
But it seems getting the status is free, so it could be the default action :) 

> get_progress_update is the pythonic naming convention

Yep, I'll fix that :)